ibm_catalog.json

```diff
@@ -69,59 +61,108 @@
                   "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-secrets-manager-public-cert-engine/main/reference-architecture/secrets_manager_public_cert_engine.svg",
```

Comment:
Not an issue as such but an optional change if you can please rename the architecture diagram from `secrets_manager_public_cert_engine.svg` to `deployable-architecture-sm-public-cert-engine.svg` just to adhere to the nomenclature we follow across DAs.
If you do so, please ensure all the references are also adjusted accordingly.